### DISCUSSION TITLE
Optimisation

### DISCUSSION THREAD

**Discussion by YiitCan** [danielkrupinski/Osiris#3679] [Ideas]
I use a GTX 980 TI it's a GPU better than 1070 The second I Inject this to my Game it near cuts in half my frames. This cheat needs optimisation.

**Comment by TheMrOrder** [comment]
Your PC problem, my performance is the same as not injecting osiris.

**Reply by YiitCan** [reply]
I have a 3080 TI and 11900K my guy it isn't my issue it's supposed to be over 700 in any situation after infection it goes under 540
